rfc-119-merge-mainstream-browse-topics-and-taxons.md

```diff
@@ -0,0 +1,183 @@
+# RFC 119: Merge mainstream browse, specialist topic into to taxon content items
+
+## Summary
+
+Mainstream browse, specialist topic and taxon content items are functionally equivalent, in that they represent a category that groups content items. We would like to maintain future consistency between our taxonomy, mainstream browse and specialist topic titles. We propose merging the attributes of mainstream browse and topic content items into taxons and deprecating use of mainstream browse and specialist topics in the Publishing API and Content Store.
+
+## Background
+
+There are currently three different and independent ways to categorise content items - [mainstream browse pages][mainstream-browse-page-docs], [specialist topics][specialist-topic-docs] and [taxons][taxon-docs]. A content item can be associated with one or more mainstream browse pages, specialist topics or taxons. Each of which represents a grouping of content items that are related in subject matter. For example content items representing a "Income Tax" guidance page and a "Inheritance Tax" guidance page are assigned to the "Tax" mainstream browse page.
+
+Mainstream browse pages, specialist topics and taxons themselves are represented as content items. Each type can contain links to other items of the same type and usually do so building a hierarchical tree structure. For example, the "Child benefit" and "Universal Credit" specialist topics are linked and represented as children of the "Benefits" specialist topic. For each type there is one "root" item that represents the start of the hierarchy. There is an exception for taxons, where there are two hierarchies with two distinct roots and sets of items - [topic taxonomy][topic-taxonomy-docs] and [world taxonomy][world-taxonomy-docs]. For purposes of this RFC ignore the world taxonomy.
+
+
+## Problem
+
+We have to maintain three separate types of categorisation for content items - mainstream browse pages, specialist topics and taxons. This means the categories themselves and the content assigned to them may be different between each type. For example, the "Education and learning" mainstream browse page is the logical equivalent of the "Education, training and skills" taxon and should be the same, however they differ in name and in the set of content items they represent. There are also categories that only exist in some types and not others, for example the "Setting up a business" mainstream browse page has no equivalent taxon.
+
+Examples of equivalent content items:
+
+Mainstream browse page: https://www.gov.uk/api/content/browse/benefits/universal-credit
+
+Specialist topic: https://www.gov.uk/api/content/topic/benefits-credits/universal-credit
```

Comment:
I feel uneasy about work going into keeping these specialist topics as a distinct thing. It seems that this was one of the main things the topic taxonomy was meant to replace it feels like we should consolidate them and topic taxonomy pages rather than keep them. As I understand, outside of the hierachy, the main difference to the topic taxonomy is that we let departments curate the links of these. Could we consider how these could become a single page?

It seems a strange to me that we have https://www.gov.uk/topic/personal-tax/self-assessment and https://www.gov.uk/money/self-assessment that are essentially the same thing (and would become the same content item) but they show different things. 

If consolidation is too much work I'd have thought it be better to just have a link between the specialist topic page and the taxon than merge them as at least then they are clearly separate things.